WordPress/Classes/ViewRelated/Support/SupportTableViewController.swift

```diff
@@ -123,15 +124,15 @@ private extension SupportTableViewController {
 
         // Help Section
         var helpSectionRows = [ImmuTableRow]()
-        helpSectionRows.append(HelpRow(title: LocalizedText.wpHelpCenter, action: helpCenterSelected()))
+        helpSectionRows.append(HelpRow(title: LocalizedText.wpHelpCenter, action: helpCenterSelected(), accessibilityIdentifier: "help-center-link-button"))
 
         if ZendeskUtils.zendeskEnabled {
-            helpSectionRows.append(HelpRow(title: LocalizedText.contactUs, action: contactUsSelected()))
-            helpSectionRows.append(HelpRow(title: LocalizedText.myTickets, action: myTicketsSelected(), showIndicator: ZendeskUtils.showSupportNotificationIndicator))
+            helpSectionRows.append(HelpRow(title: LocalizedText.contactUs, action: contactUsSelected(), accessibilityIdentifier: "contact-support-button"))
+            helpSectionRows.append(HelpRow(title: LocalizedText.myTickets, action: myTicketsSelected(), showIndicator: ZendeskUtils.showSupportNotificationIndicator, accessibilityIdentifier: "my-tickets-button"))
             helpSectionRows.append(SupportEmailRow(title: LocalizedText.contactEmail,
                                                    value: ZendeskUtils.userSupportEmail() ?? LocalizedText.emailNotSet,
                                                    accessibilityHint: LocalizedText.contactEmailAccessibilityHint,
-                                                   action: supportEmailSelected()))
+                                                   action: supportEmailSelected(), accessibilityIdentifier: "set-contact-email-button"))
```

Comment:
Nitpick. The rest of the arguments are defined on new lines, this new code should be consistent with that style.

```suggestion
                                                   action: supportEmailSelected(),
                                                   accessibilityIdentifier: "set-contact-email-button"))
```

As an aside, I find it useful to break long parameters list this way, because it removes the need for scrolling back and forth horizontally.

For example, to see the rest of the code at line 131, I need to scroll horizontally

![image](https://user-images.githubusercontent.com/1218433/125237945-d57f8300-e329-11eb-9f47-53d8e03d95a6.png)

But once I've done that, I can't see the code in the other lines, so I need to scroll back horizontally

![image](https://user-images.githubusercontent.com/1218433/125237989-e9c38000-e329-11eb-983a-63eac517e700.png)

Reply:
More readable, thanks!